### ISSUE TITLE
no such column: zalttext

### ISSUE THREAD

**Issue by yuletide** [sballin/alfred-search-notes-app#44]
**To Reproduce**

Steps/context to reproduce the behavior:
Just updated to 3.4.0 and I now get this error when typing "n ", i never get the permission error nor see the "allow" button in security. 

1. Search for a note
2. ...

**no such column: zalttext**

```


Please copy this entire text if submitting an error report. You may need to scroll.

panic: no such column: zalttext

goroutine 1 [running]:
main.PanicOnErr(...)
	github.com/sballin/alfred-search-notes-app/main.go:440
main.main()
	github.com/sballin/alfred-search-notes-app/main.go:453 +0x7f3

macOS: 10.15.7
Alfred: 4.3.4
Workflow: 3.4.0
CPU: x86_64h
Binary xattrs: 
Notes DB xattrs: com.apple.quarantine
Binary info: 0755 502/ay 20/staff
Notes DB info: 0644 502/ay 20/staff
Query: title ''
Accounts: iCloud, Mapbox
```

Copy and paste the full error text from any error messages. Screenshots can also help but may not show the full text.

**Info**

Fill out this information if it's not already included in the error text.

 - In System Preferences > Security & Privacy > Privacy, there are checkmarks for Accessibility > Alfred 4, Full Disk Access > Alfred 4, Automation > Alfred 4 > Notes, and Automation > Alfred 4 > System Events? - yes
 - What type of account are your notes stored in (iCloud/On My Mac/Email)? - iCloud, Google (error still happens when this is disabled)
 - If the workflow can't manage to show a list of your notes, verify that the search_notes executable file is present by right-clicking the "Search Notes" workflow in the workflow list of Alfred Preferences, choosing "Open in Finder" and opening the search_notes folder. Is there a file of the same name inside that folder? - yes
 - If no error notification appears automatically, what is the output of the [debug log](https://www.alfredapp.com/help/workflows/advanced/debugger/) when you reproduce the problem?
```
[20:02:46.725] Logging Started...
[20:02:49.926] Search Notes[Script Filter] Queuing argument ''
```

**Comment by sballin** [comment]
Thanks for the quick error report. I overlooked the differences in the Notes database between macOS 10.15 and macOS 11, so for now the solution would be to either upgrade macOS or downgrade the workflow version to 3.3.0. I'll try to fix this in the code...

**Comment by autumnfjeld** [comment]
I get the same error.

**Comment by yuletide** [comment]
Sounds good, for anyone else the [downgrade was easy](https://github.com/sballin/alfred-search-notes-app/releases/download/3.3.0/Search.Notes.alfredworkflow) and fixed things

**Comment by sballin** [comment]
Good to know that it also happens on macOS 11.1... Looks like I wrote code that works specifically on macOS >=11.4

**Comment by sballin** [comment]
Ok I've released 3.4.1 which should fix this error for everyone.

**Comment by autumnfjeld** [comment]
That was fast!!! ![bitmoji](https://sdk.bitmoji.com/render/panel/469d4aea-0aaa-4b89-919e-0fd853b6f0f0-b931236c-c79a-49b5-b0f5-ab26d7e4cd09-v1.png?transparent=1&palette=1&width=46)
